<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>18</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <widget class="TitleLabel" name="TitleLabel">
     <property name="text">
      <string>时间线编辑</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="SubtitleLabel" name="SubtitleLabel_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>33</height>
          </size>
         </property>
         <property name="text">
          <string>节点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CaptionLabel" name="tips_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">FluentLabelBase {
    color: black;
}

HyperlinkLabel {
    color: #009faa;
    border: none;
    background-color: transparent;
    text-align: left;
    padding: 0;
    margin: 0;
}

HyperlinkLabel[underline=true] {
    text-decoration: underline;
}

HyperlinkLabel[underline=false] {
    text-decoration: none;
}

HyperlinkLabel:hover {
    color: #007780;
}

HyperlinkLabel:pressed {
    color: #00a7b3;
}
FluentLabelBase{color:#7d000000}</string>
         </property>
         <property name="text">
          <string>还未添加任何节点</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
         <property name="lightColor" stdset="0">
          <color alpha="125">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="darkColor" stdset="0">
          <color alpha="155">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="pixelFontSize" stdset="0">
          <number>14</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ListWidget" name="part_list">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="BodyLabel" name="BodyLabel_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>节点名称</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="EditableComboBox" name="name_part_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="BodyLabel" name="BodyLabel_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>开始时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TimeEdit" name="part_time">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>33</height>
            </size>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::HourSection</enum>
           </property>
           <property name="displayFormat">
            <string>h:mm</string>
           </property>
           <property name="time">
            <time>
             <hour>7</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ToolButton" name="add_part_button"/>
         </item>
         <item>
          <widget class="ToolButton" name="delete_part_button"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="VerticalSeparator" name="VerticalSeparator"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="SubtitleLabel" name="SubtitleLabel">
           <property name="text">
            <string>时间线</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ComboBox" name="select_timeline">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="CaptionLabel" name="tips_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">FluentLabelBase {
    color: black;
}

HyperlinkLabel {
    color: #009faa;
    border: none;
    background-color: transparent;
    text-align: left;
    padding: 0;
    margin: 0;
}

HyperlinkLabel[underline=true] {
    text-decoration: underline;
}

HyperlinkLabel[underline=false] {
    text-decoration: none;
}

HyperlinkLabel:hover {
    color: #007780;
}

HyperlinkLabel:pressed {
    color: #00a7b3;
}
FluentLabelBase{color:#7d000000}</string>
         </property>
         <property name="text">
          <string>还未添加任何时间线</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
         <property name="lightColor" stdset="0">
          <color alpha="125">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="darkColor" stdset="0">
          <color alpha="155">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="pixelFontSize" stdset="0">
          <number>14</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ListWidget" name="timeline_list">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="BodyLabel" name="BodyLabel_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>活动类型</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ComboBox" name="class_activity">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BodyLabel" name="BodyLabel_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>时段</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ComboBox" name="time_period">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="BodyLabel" name="BodyLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>时长（分钟）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SpinBox" name="spin_time">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ToolButton" name="add_button"/>
         </item>
         <item>
          <widget class="ToolButton" name="edit_button"/>
         </item>
         <item>
          <widget class="ToolButton" name="delete_button"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PrimaryPushButton" name="save">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>EditableComboBox</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>VerticalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
